index.js

```diff
@@ -40,19 +43,19 @@ app.on('sticker', (ctx) => {
         console.log("write");
       }
       console.log("open");
-      ctx.reply('Club abierto')
+      ctx.reply(dict['6'])
     }
     else if (ctx.message.sticker.emoji == '😒' && ctx.message.sticker.set_name == "CRUPC") {
       if (file == "open") {
         fs.writeFileSync("stateCRUPC", "closed");
         console.log("write");
       }
       console.log("closed");
-      ctx.reply('Club cerrado')
+      ctx.reply(dict['7'])
     }
   } 
   else {
-    ctx.reply('A ti no te hago ni puto caso');
+    ctx.reply(dict['8']);
```

Comment:
This reply is the same as the number 5

Reply:
Fixed, my friend. Any other things before merging?